react/NestedSelect/NestedSelect.jsx

```diff
@@ -1,9 +1,10 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { withStyles } from '@material-ui/core'
-import Icon from '../Icon'
-import UIRadio from '../Radio'
 import omit from 'lodash/omit'
+
+import Icon from '../Icon'
+import Radio from '../Radios'
```

Comment:
nit: we could check there is a radio used in the test  of NestedSelect

Replacing this import by () => <div/> does not fail test... 😢

Reply:
J'ai plutôt corrigé le test